drivers/include/tsl4531x.h

```diff
@@ -96,7 +96,8 @@ typedef struct {
     i2c_t i2c_dev;                              /**< I2C device which is used */
     i2c_t i2c_addr;                             /**< I2C address of sensor */
     tsl4531x_intgn_time_t integration_time;     /**< integration time */
```

Comment:
If you are going to use bitfields, why not make `tsl4531x_intgn_time_t` a bitfield of size 2 and save 4 bytes for free?

Reply:
I implemented this and looked at the build sizes - it actually doesn't save any space in RAM but adds 32 bytes to flash because of the instructions required to handle the bitfields